runtime/assembly/__tests__/mock-builder.ts

```diff
@@ -12,85 +14,143 @@ export namespace MockBuilder {
+    /**
+     * Returns SCALE Encoded Header with Digests Mock and Instance of that Header
+     */
+    export function getHeaderWithDigestsMock(): MockResult<Header> {
+        return new MockResult(MockHelper.getHeaderInstanceWithDigests(), MockConstants.HEADER_WITH_DIGEST);
     }
 
     /**
      * Returns SCALE encoded Inherent Mock and Instance of that Mock
      */
-
      export function getInherentMock(): MockResult<Inherent> {
-        const DEFAULT_INHERENT: u8[] = [
-            0, 0, 0, 0, 0, 0, 0, 0, 
-            0, 0, 0, 0, 0, 0, 0, 0,
-            0
-        ];
-
-        return new MockResult(MockHelper._getEmptyInherentInstance(), DEFAULT_INHERENT);
+         return new MockResult(MockHelper.getEmptyInherentInstance(), MockConstants.DEFAULT_INHERENT);
      }
 
+    /**
+     * Returns SCALE encoded extrinsic mock and Instance of that mock
+     */
     export function getDefaultExtrinsic(): MockResult<Extrinsic> {
-        const DEFAULT_EXTRINSIC: u8[] = [
-            1, 212, 53, 147, 199, 21, 253, 211, 28, 97, 20, 26, 189, 4, 169, 159, 214, 130, 44, 133, 88, 133, 76, 205, 227, 154, 86, 132, 231, 165, 109, 162,
-            125, 142, 175, 4, 21, 22, 135, 115, 99, 38, 201, 254, 161, 126, 37, 252, 82, 135, 97, 54, 147, 201, 18, 144, 156, 178, 38, 170, 71, 148, 242, 106,
-            69, 0, 0, 0, 0, 0, 0, 0,
-            5, 0, 0, 0, 0, 0, 0, 0,
-            72, 43, 234, 45, 159, 200, 43, 162, 117, 34, 73, 0, 41, 24, 219, 106, 202, 41, 220, 128, 114, 102, 33, 40, 235, 200, 34, 98, 249, 135, 134, 116, 39, 94, 159, 122, 148, 102, 158, 5, 178, 195, 144, 165, 149, 149, 118, 250, 97, 192, 228, 0, 216, 37, 219, 207, 7, 240, 82, 75, 243, 191, 237, 138, 0
-        ];
-        return new MockResult(MockHelper._getExtrinsicInstance(), DEFAULT_EXTRINSIC);
+        return new MockResult(MockHelper.getExtrinsicInstance1(), MockConstants.DEFAULT_EXTRINSIC);
     }
 
+    /**
+     * Returns invalid SCALE Encoded extrinsic
+     */
     export function getInvalidExtrinsic(): u8[] {
         // the minimum length of the input should be 144
-        const INVALID_EXTRINSIC: u8[] = [
+        return [
             37, 219, 207, 7, 240, 82, 75, 243, 191, 237, 138, 0
         ];
-        return INVALID_EXTRINSIC;
+    }
+
+    /**
+     * Returns SCALE Encoded DigestItem with type Option and Instance of that Digest Item
+     */
+    export function getOtherDigestItemMock(): MockResult<DigestItem> {
+        const otherDigestValue = ByteArray.fromU8a([12, 1, 1, 1]);
+        return new MockResult(new Other(otherDigestValue), MockConstants.OTHER_DIGEST);
+    }
+
+    /**
+     * Returns SCALE Encoded ChangeTrieRoot and Instance of that Digest Item
+     */
+    export function getChangeTrieRootDigestItemMock(): MockResult<DigestItem> {
+        const changeTrieRoot = Hash.fromU8a([255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255]);
```

Comment:
Maybe create a utility function to create Hash populated with a number (for example here, it's 255)?